src/bag3_digital/schematic/inv.py

```diff
@@ -68,7 +69,7 @@ def get_default_param_values(cls) -> Dict[str, Any]:
 
     def design(self, seg: int, seg_p: int, seg_n: int, lch: int, w_p: int, w_n: int, th_p: str,
                th_n: str, stack_p: int, stack_n: int, p_in_gate_numbers: Optional[List[int]] = None,
-               n_in_gate_numbers: Optional[List[int]] = None) -> None:
+               n_in_gate_numbers: Optional[List[int]] = None, vertical_out: bool = True) -> None:
```

Comment:
This is absolutely not how BAG schematic generators are written. The default value should go in get_default_params_values(), not here.

Reply:
You should take this up against the `develop` branch then because this change comes from a merge conflict resolution against the existing version

Reply:
Look at lines 77-78 in `schematic/inv.py` on the develop branch

Reply:
oh never mind I see what you mean